onnx/test/cpp/schema_test.cc

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <iostream>
+#include "gtest/gtest.h"
+#include "onnx/defs/operator_sets.h"
+#include "onnx/defs/schema.h"
+
+using namespace ONNX_NAMESPACE;
+
+namespace ONNX_NAMESPACE {
+namespace Test {
+
+TEST(SchemaTest, RegisterCertainOpsetSchema) {
+#ifdef __ONNX_DISABLE_STATIC_REGISTRATION
```

Comment:
add some comments explaining why this test is run only when static registration is disabled